<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager</class>
 <widget class="QDialog" name="manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>经理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-image: url(:/image1/图片/38809f6c95008063.jpg);
}

QPushButton{
background-color: rgb(85, 255, 255);	}

font: 75 12pt &quot;楷体&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;
background-color: rgb(25, 136, 255);</string>
       </property>
       <property name="text">
        <string>服务员</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;
background-color: rgb(25, 136, 255);</string>
       </property>
       <property name="text">
        <string>厨师</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QStackedWidget{
	background-image: url(:/image1/图片/b467ec84f3d6ecb6-d09a660dba031690-9f5174a42ab4a04620d2ad7b88370212.jpg);}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="styleSheet">
          <string notr="true">QTableView{
	border-image: url(:/image1/图片/d84df8aacf4e21bd.jpg);}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="tableView_2">
         <property name="styleSheet">
          <string notr="true">QTableView{
	background-image: url(:/image1/图片/636f0b10610705a3.jpg);}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);
font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
